libzmix/bulletproofs_amcl/src/r1cs/gadgets/set_membership_alt.rs

```diff
@@ -0,0 +1,227 @@
+use super::helper_constraints::constrain_lc_with_scalar;
```

Comment:
Is there a reason to include this? Is it any more efficient than `set_membership`

Reply:
These gadgets have not had much peer review. I did not want to discard this one as i took it from another source. The one in `set_membership` is efficient but i have not seen that being used anywhere.